racingcar-java/src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,80 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class CalculatorTest {
+
+    private Calculator calculator = new Calculator();
+
+    @Test
+    @DisplayName("덧셈")
+    void 문자열계산기_덧셈() {
```

Comment:
한가지 tmi를 드리자면 DisplayName은 junit5때부터 생긴 어노테이션이에요.
void 문자열계산기_덧셈()처럼 만드신 방식은 한글로 이를 표현하고 싶었지만 차마 그런 기능이 없어서 junit4에서 작성하던 방식인거죠.

`@DisplayName`을 사용함으로써 테스트를 진행할 때 이름을 한글로 표현하고, 메소드 명을 영어로 진행하는 것이 junit5에서 테스트를 작성하는 방식이에요 :)
```suggestion
    void calculatorPlusTest() {
```